publiccode.yml

```diff
@@ -0,0 +1,99 @@
+landingURL: https://www.interop.pagopa.it/
+releaseDate: 2025-08-07
+developmentStatus: stable
+softwareType: standalone/web
+platforms:
+  - web
+categories:
+  - resource-management
+  - application-development
+  - business-intelligence
+  - collaboration
+usedBy:
+  - Dipartimento per la Trasformazione Digitale - Presidenza del Consiglio dei
+    Ministri
+  - PagoPA S.p.A.
+description:
+  it:
+    localisedName: Piattaforma Digitale Nazionale Dati
+    shortDescription: La Piattaforma Digitale Nazionale Dati (PDND) abilita
+      l'interoperabilità dei sistemi informativi degli Enti e dei Gestori di
+      Servizi Pubblici.
+    longDescription: >-
+      ## Vision del progetto
+
+      La Piattaforma Digitale Nazionale Dati (PDND) abilita l'interoperabilità
+      dei sistemi informativi degli Enti e dei Gestori di Servizi Pubblici,
+      rendendo concreto il principio "once-only". Gli aderenti alla piattaforma
+      potranno comunicare tra loro in modo semplice, veloce e sicuro, senza
+      dover più chiedere ai cittadini informazioni già in possesso di altri
+      enti.
+
+
+      ## Vantaggi
+
+      ### Per i Cittadini
+
+      Cittadini e imprese non dovranno più fornire le informazioni che gli enti
+      già possiedono, ogni volta che avranno necessità di accedere a un
+      servizio. Questo avrà l'effetto principale di semplificare e velocizzare i
+      tempi di fruizione dei servizi pubblici, garantendo la sicurezza dei dati
+      degli interessati.
+
+
+      ### Per il Paese
+
+      Grazie alla standardizzazione dei processi, la Piattaforma permette agli
+      enti di scambiare informazioni in maniera sicura e immediata e ne
+      favorisce il riutilizzo da parte di tutti gli attori coinvolti. Questo
+      abilita una collaborazione virtuosa e qualificata tra i diversi enti e
+      istituzioni consentendo loro di erogare servizi pubblici in modo più
+      veloce ed efficiente.
+
+      ### Informazioni aggiuntive
+
+      Il prodotto nella sua interezza è licenziato con una licenza EUPL, versione 1.2. 
+      Nota bene: il progetto software è organizzato con una struttura a monorepo. 
+      Pertanto, esplorando il repository è possibile esplorare i singoli "package", ognuno contenente le informazioni di licensing e autorship dedicate.
+
+    documentation: "https://docs.pagopa.it/interoperabilita-1"
+    apiDocumentation: "https://docs.pagopa.it/interoperabilita-1/manuale-operativo/api-esposte-da-pdnd-interoperabilita"
+
+    features:
+      - Facilita l'integrazione di servizi via API
+      - Permette lo scambio di informazioni tra enti
+      - Facilita le operazioni di compliance grazie ad interfacce standard
+legal:
+  license: EUPL-1.2
+  mainCopyrightOwner: Dipartimento per la Trasformazione Digitale - Presidenza del Consiglio dei Ministri
+  repoOwner: PagoPA S.p.A.
+maintenance:
+  type: internal
+  contacts:
+    - name: "Open Source Program Office"
+      email: "ospo@pagopa.it"
```

Comment:
is this the contact mail for the maintenance of this file or the code of the platform?

Reply:
That's the entry point for general purpose "open source" questions coming from outside. I'd keep it as it is.